### PR TITLE
Adds support for SMS-related fields.

### PR DESCRIPTION
**Pull request by DFurnes**
### Changes
This pull request adds support for SMS related fields in Customer.io!

📝 Renames `mobile_status` to `sms_status` to be consistent with naming in Northstar. Since we haven't been sending phone numbers to Customer.io, this should still be safe to change. (24fa2a0)

☎️ Adds support for the E.164-formatted `mobile` field. (be8c04c)

### How should this be reviewed?
Let me know if there are any things I'm missing!

### CODE REVIEW

File: test/messages/CustomerIoUpdateCustomerMessage.test.js [hunk 1]
```diff
@@ -161,8 +162,8 @@ test('Cio identify created from Northsar is correct', () => {
 
     // Optional:
     if (cioUpdateAttributes.mobile_status) {
```
Comment (sergii-tkachenko) [inline test/messages/CustomerIoUpdateCustomerMessage.test.js]: Should this be `cioUpdateAttributes. sms_status` too?

Comment (DFurnes) [reply test/messages/CustomerIoUpdateCustomerMessage.test.js]: Ah yes! Good catch. 🎣

**Comment by DFurnes** [comment]
Okay, got 'em! It'd be nice to look into making a factory method for creating these test users in the future, so schema changes don't involve so much find-and-replace. 🔍

**Comment by DFurnes** [comment]
Caught one more piece in a2c25fb. Ran through some additional manual testing of this on my local & it looks like the correct payloads are being sent to Customer.io for `phone` and `sms_status`.

@sergii-tkachenko Ready for a re-review at your convenience!

**Comment by sergii-tkachenko** [comment]
Tested on staging. Looks great!

**Pull request merged**